project/login.html

```diff
@@ -130,21 +130,40 @@
 				margin-left: 1400px;
 			}
 
+			::-webkit-input-placeholder { /* WebKit, Blink, Edge */
```

Comment:
Weeeell, maybe not really. :P That differs from project to project, but most of the projects you'll be working on in your professional career will likely support only modern browser versions. You're the lucky ones, in this sense: you don't need to care all that much about differences in browser capabilities.